### PR TITLE
Add cards from Servants of Strife preview.

### PR DESCRIPTION
**Pull request by FreakyDug**

### CODE REVIEW

File: data/pilots/separatist-alliance/belbullab-22-starfighter.json [hunk 1]
```diff
@@ -24,7 +24,99 @@
     "5FW"
   ],
   "faction": "Separatist Alliance",
-  "stats": [{ "type": "shield", "value": 2 }],
-  "actions": [],
-  "pilots": []
+  "stats": [
+    { "type": "attack", "value": 2, "arc": "Front Arc" },
```
Comment (guidokessels) [inline data/pilots/separatist-alliance/belbullab-22-starfighter.json]: Attack value should be `3`

Comment (FreakyDug) [reply data/pilots/separatist-alliance/belbullab-22-starfighter.json]: Woops

File: data/pilots/separatist-alliance/belbullab-22-starfighter.json [hunk 2]
```diff
@@ -24,7 +24,99 @@
...
+      "image": "https://images-cdn.fantasyflightgames.com/filer_public/5e/3d/5e3d8e36-3989-40f4-9908-6bd6583bb88a/swz29_wat-tambor.png"
+    },
+    {
+      "name": "Feethan Ottraw Autopilot",
+      "limted": 0,
+      "initiative": 1,
+      "xws": "feethanottrawautopilot",
+      "text": "Unlike the more disposable fighters it also built for the Separatists, Feethan Ottraw Scalable Assemblies designed the Belbullab-22 with a solid mix of firepower, durability, and speed.",
+      "image": "https://images-cdn.fantasyflightgames.com/filer_public/6e/76/6e7626dc-3b81-4290-aece-ddf5d86e7667/swz29_autopilot.png",
+      "actions": [
```
Comment (guidokessels) [inline data/pilots/separatist-alliance/belbullab-22-starfighter.json]: This should be `shipActions`

**Comment by guidokessels** [comment]
Could you add `hyperspace: false` to each pilot? (you can use `npm run hyperspace` to add it)

**Pull request merged**